### ISSUE TITLE
A11y_Code.visualstudio.com_Visual studio Search page_Search text field_High Contrast: In high contrast black mode the "Search" icon is not visible.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5153]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG4.3.1;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;
## Environmental Details:
#URL: https://code.visualstudio.com/Search
## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version	10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)
## Pre-Requisite:
To apply high contrast follow the below steps:

1. Open settings, tab till Ease of access and press enter.
2. Tab till High contrast press enter.
3. From the drop down select black mode.
4. Tab till apply button and press enter
## Repro Steps :
1. Launch the URL https://code.visualstudio.com/Search
2. Tab till "Search" icon.
3. Observe the "Search" icon is adopting black contrast mode or not.

## Actual Result :
In high contrast black mode the "Search" icon is not visible.
## Expected Result:
In high contrast black mode all the  icons should be properly visible.
## User Impact :
Low vision users  may find it difficult because they can't able to  see the icon properly.
## Attachments : 
![Bug_5153_Visual studio Search page_Search text field_High Contrast](https://user-images.githubusercontent.com/99243079/153132118-93048376-e3b6-480a-83e7-4d1dc0f743a9.png)

**Comment by Samhitha66** [comment]
![Bug_5153_Visual studio Search page_Search text field_High Contrast](https://user-images.githubusercontent.com/99243079/153124570-6a6022a6-23f5-4765-a667-5f65ac60c53f.png)

**Comment by gregvanl** [comment]
The current fix made it so the search icon is no longer visible when not in High Contrast Mode.

![image](https://user-images.githubusercontent.com/12818376/153511418-61109747-5c12-4411-808f-b6ef7009d4f4.png)

**Comment by rzhao271** [comment]
Reverted the change. I need to refactor the CSS around the search icon first.

**Comment by Samhitha66** [comment]
Checked again this bug in the below environment 
URL: https://code.visualstudio.com/Search

Now we can be able to see the search icon in high contrast black mode.
So, the issue is fixed. 
![Bug_5153](https://user-images.githubusercontent.com/99243079/157193314-099b593a-21b0-4632-9663-c8358536d0bd.png)
